types/address_test.go

```diff
@@ -9,14 +9,23 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
 	"github.com/tendermint/tendermint/crypto"
-	yaml "gopkg.in/yaml.v2"
+	"gopkg.in/yaml.v2"
 
 	"github.com/cosmos/cosmos-sdk/crypto/keys/ed25519"
 	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
 	"github.com/cosmos/cosmos-sdk/types"
 )
 
+type AddressTestSuite struct {
+	suite.Suite
+}
+
+func TestUnittestTestSuite(t *testing.T) {
+	suite.Run(t, new(AddressTestSuite))
+}
```

Comment:
:+1:

Reply:
BTW - we can have one `TestSuite` function which will instantiate all test suites:

```
func TestSuite(t *testing.T) {
	suite.Run(t, new(AddressTests))
        suite.Run(t, new(IntegrationTests))
        ...
}
```
Then usually in `Setup` function I prefer to test flags:
```
if t.Short() {
    t.Skip()
}
```
We should add probably own flags, eg: `-t.unit`

Reply:
No thanks. Let's use `-short` to disable integration testing execution only. Introducing a flag for each distinct class of tests (i.e. `unit` VS `integration`) IMHO 1. feels an overkill 2. defeat the point of this developer experience-focused effort (when it comes to DevX and flags, the more the flags, the worse the outcome [1])

[1] This is too quite Golang idiomatic.

Reply:
OK, if we don't need other flags, then yes - better to not add them.

Reply:
Well, maybe at some point we'll need more flags! In that case, we'll add them :+1: